tests/src/test/java/alluxio/client/rest/RestApiTest.java

```diff
@@ -13,17 +13,14 @@
 
 import alluxio.testutils.BaseIntegrationTest;
 
-import java.util.HashMap;
+import com.google.common.collect.ImmutableMap;
+
 import java.util.Map;
 
 public abstract class RestApiTest extends BaseIntegrationTest {
-  protected static final Map<String, String> NO_PARAMS = new HashMap<>();
+  protected static final Map<String, String> NO_PARAMS = ImmutableMap.of();
 
   protected String mHostname;
   protected int mPort;
   protected String mServicePrefix;
-
-  protected String getEndpoint(String suffix) {
-    return mServicePrefix + "/" + suffix;
-  }
```

Comment:
This method is being removed as it encouraged the use of manually prefixing the `endpoint` string with the REST API service prefix, as opposed to relying on the `mPrefix` field in `TestCase.java`.

Reply:
`mServicePrefix` is being removed in favour of `mBaseUri` (which was previously implicit`. If a prefix is required, callers should overwrite `mBaseUri` with the new base URI.